validator/server_arb/validator_spawner.go

```diff
@@ -244,7 +244,12 @@ func (v *ArbitratorSpawner) CreateExecutionRun(wasmModuleRoot common.Hash, input
 			machine.Destroy()
 			return nil, err
 		}
-		wrapped := MachineInterface(machine)
+		var wrapped MachineInterface
+		if *useBoldMachine {
```

Comment:
This should be `if useBoldMachine != nil && *useBoldMachine`, as legacy clients won't supply that parameter.